enable/wx/base_window.py

```diff
@@ -352,26 +354,31 @@ def _create_mouse_event ( self, event ):
             x           = event.GetX()
             y           = event.GetY()
             self._last_mouse_pos = (x, y)
-            mouse_wheel = ((event.GetLinesPerAction() *
+
+            mouse_wheel = ((float(event.GetLinesPerAction()) *
```

Comment:
So do the calculation in floating point and suddenly you match the magnitude of Qt wheel events? Subtle.